var/spack/repos/builtin/packages/lbann/package.py

```diff
@@ -272,15 +287,36 @@ def initconfig_compiler_entries(self):
             # of CMake
             entries.append(cmake_cache_option("CMAKE_EXPORT_COMPILE_COMMANDS", True))
 
+        entries.append(cmake_cache_string("CMAKE_INSTALL_RPATH_USE_LINK_PATH", "ON"))
+        linker_flags = "-Wl,--disable-new-dtags"
```

Comment:
(Only blocking question) Will this setting conflict with `config:shared_linking:type:runpath`?

Reply:
@alalazo this flag `entries.append(cmake_cache_string("CMAKE_INSTALL_RPATH_USE_LINK_PATH", "ON"))` matches what is in the `cmake` package.  The  `linker_flags = "-Wl,--disable-new-dtags"` is used to make sure that `RPATH` works properly on all of the systems that we are working with.

Reply:
It's just the linker flag I'm concerned about. It's a bad idea to mix RPATH and RUNPATHs, and if a user sets `config:shared_linking:type:runpath` there will be RUNPATH set on dependencies. Do you know why the flag was necessary? That flag is supposed to be injected in our compiler wrapper based on the config value.